examples/serve-simple-v1alpha2.yml

```diff
@@ -1,27 +0,0 @@
-apiVersion: "machinelearning.seldon.io/v1alpha2"
```

Comment:
In the CRD, we're still claiming that we support `v1alpha2`. Might be worth keeping this around to ensure that's true.

Reply:
I couldn't find what you are saying in the CRD, do you mind pointing me there? If we are still supporting `v1alpha2`, sure we can keep it for testing in the future.

Reply:
@DnPlas Here's where the CRD defines the default of `v1`, but also some older versions, including `v1alpha2`:

https://github.com/canonical/seldon-core-operator/blob/cf8f0e61/src/templates/crd.yaml#L4425-L4435

Reply:
Oh, sweet. I added it back. Thanks!